### ISSUE TITLE
WCAG issue: Theming - My Media Publish dialog is not keyboard or screen reader accessible

### ISSUE THREAD

**Issue by Rainribbon** [Kaltura-EDU/issues-tracking#23]
Submitted to Kaltura as Case #00549538:

With Theming enabled, I am unable to use the My Media Publish dialog with a keyboard. When tabbing to the Actions menu for an individual media entry and opening the Publish dialog, the dialog does not capture the keyboard successfully. Pressing Tab closes the dialog.

Additionally, the Publish menu that appears at the top of the screen after selecting media entries with checkboxes cannot be used with a keyboard.

Because these elements are inaccessible to keyboard users, they are also inaccessible to users of other assistive technology such as screen reader users.

This is a failure of the following WCAG Success Criteria:
* [SC 2.1.1 Keyboard (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)
* [SC 2.1.2: No Keyboard Trap (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/no-keyboard-trap.html)

Steps to reproduce:
1. In a KAF with theming enabled, have or upload several media entries to My Media. Also, be enrolled in at least two courses as an instructor.
2. Use the Tab key on the keyboard to navigate to the Actions menu for a media entry. Press Enter to open the menu.
3. Use the down arrow key to navigate to Publish Status, then right arrow > down arrow to navigate to Publish.
4. Press Enter to open the Publish dialog.
5. Press Tab to try to navigate the Publish dialog.
** Expected behavior: Your focus goes to the checkbox for the first course in the list to be able to select it via spacebar, and/or Tab to the next course in the list.
** Actual behavior: Dialog window closes.
6. Tab over to a checkbox for a media entry and press spacebar to select it.
7. Tab to another checkbox for a media entry and press spacebar to select it.
8. Shift + Tab up to the Publish menu that appears after selecting entries.
9. Press Enter.
10. Try to interact with the menu.
** Expected behavior: You can arrow down to focus on either Private or Publish, and press Enter to either make the media private (if Private is in focus) or open the Publish dialog (if Publish is in focus).
** Actual behavior: The only functional key is Escape to close the menu.

**Comment by Rainribbon** [comment]
Kaltura is tracking this under SUP-48608 | My Media Publish Dialog and Top Publish Menu Inaccessible via Keyboard with Theming Enabled.
